app/code/Magento/Quote/Model/Quote/Address/Validator.php

```diff
@@ -48,6 +48,18 @@ public function isValid($value)
             $messages['invalid_email_format'] = 'Invalid email format';
         }
 
+        $pattern_name = '/(?:[\p{L}\p{M}\,\-\_\.\'’`\s\d]){1,255}+/u';
+        $messages = $this->isValidPattern($pattern_name, $value->getFirstname(), $messages, 'invalid_firstname', 'First Name is not valid!');
+        $messages = $this->isValidPattern($pattern_name, $value->getLastname(), $messages, 'invalid_lastname', 'Last Name is not valid!');
+        $messages = $this->isValidPattern($pattern_name, $value->getMiddlename(), $messages, 'invalid_middlename', 'Middle Name is not valid!');
+        $messages = $this->isValidPattern($pattern_name, $value->getCompany(), $messages, 'invalid_company', 'Company is not valid!');
+        $pattern_street = "/(?:[\p{L}\p{M}\"[],-.'’`&\s\d]){1,255}+/u";
+        foreach ($value->getStreet() as $street) {
+            $messages = $this->isValidPattern($pattern_street, $street, $messages, 'invalid_street', 'Street is not valid!');
+        }
+        $pattern_city = '/(?:[\p{L}\p{M}\s\-\']{1,100})/u';
```

Comment:
`city` field in the database accepts 255 signs, i not see why here they will be limited to random "100" symbols.

Reply:
The same must be applied to `region` field also.

Reply:
Hola a todos:

I'm not xavi but i participate in the coding, we copied all patterns from https://github.com/magento/magento2/pull/38345, you need to point the same there